README.md

```diff
@@ -56,6 +56,7 @@ Chromeless comes out of the box with a remote proxy built-in - the usage stays c
 ## Installation
 ```sh
 npm install chromeless
+node "name-of-script.js"
```

Comment:
I prefer we omit this here because this section is about installation and not execution. Perhaps we can add it to the Usage section.